optuna/trial.py

```diff
@@ -12,18 +13,28 @@
     from typing import Dict  # NOQA
     from typing import Optional  # NOQA
     from typing import Sequence  # NOQA
+    from typing import Union  # NOQA
 
     from optuna.distributions import BaseDistribution  # NOQA
     from optuna.distributions import CategoricalChoiceType  # NOQA
     from optuna.study import Study  # NOQA
 
+    FloatingPointDistributionType = Union[
+        distributions.UniformDistribution, distributions.LogUniformDistribution
+    ]
+
 
 class BaseTrial(object, metaclass=abc.ABCMeta):
     """Base class for trials.
 
     Note that this class is not supposed to be directly accessed by library users.
     """
 
+    def suggest_float(self, name, low, high, log):
```

Comment:
How about specifying the default value of `log` argument?
```suggestion
    def suggest_float(self, name, low, high, log=False):
```

Reply:
The default values were introduced in 988bdf9.